src/main/java/org/spongepowered/api/command/CommandCallable.java

```diff
@@ -79,6 +79,20 @@
     boolean testPermission(CommandSource source);
 
     /**
+     * Test whether this command is allowed to be executed by the specified
+     * source depending on it's type.
+     *
+     * <p>Implementations should use this to restrict command execution of
+     * certain command source types. This varies from permissions in that
+     * a source may have permission to execute a command but must be, for
+     * example, a Player because the command uses locational data.</p>
+     *
+     * @param source called of the command
```

Comment:
Is this supposed to be "Caller of the command"? (Also note the first word after param name must be capitalized)